lein-kibit/project.clj

```diff
@@ -0,0 +1,9 @@
+(defproject lein-kibit "0.1.4-SNAPSHOT"
+  :description "kibit lein plugin"
+  :url "https://github.com/jonase/lein-kibit"
+  :license {:name "Eclipse Public License"
+            :url  "http://www.eclipse.org/legal/epl-v10.html"}
+  :dependencies [[jonase/kibit "0.1.4-SNAPSHOT"]
```

Comment:
So it looks like we can drop this particular dependency instance altogether. The leiningen plugin doesn't directly load any kibit code, it just drives the user's project with the matching kibit dependency. The patches below refactor the VERSION file out to a common resource tree, and load it from there (or include it in the jar) as appropriate.